tests/jballerina-unit-test/src/test/resources/test-src/taintchecking/connectors/httpclient.bal

```diff
@@ -15,10 +15,12 @@ public function main (string... args) {
         if (msg is string) {
             normalFunction(msg);
         } else {
-            panic msg;
+            error err = msg;
+            panic err;
         }
     } else {
-        panic response;
+        error err = response;
+        panic err;
```

Comment:
```suggestion
        panic <error> response;
```

Reply:
We can get rid of additional variable creation, if we do the cast and panic. WDYT?

Reply:
Agreed, done